### PR TITLE
Forbid access to unauthenticated users and users without roles

### PR DESCRIPTION
**Pull request by parndt**
Per #1, this changes the default application policy to require an
authenticated user that has any roles assigned.

This rule excludes public and guest users.

### CODE REVIEW

**Comment by parndt** [comment]
@tmfrnz ready for you to take a look

File: app/policies/page_policy.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@ def index?
     true
   end
 
+  def show?
+    true
```
Comment (tmfrnz) [inline app/policies/page_policy.rb:7]: to clarify: pages should also only be visible for users with a role assigned

**Comment by parndt** [comment]
> I assume, this does not yet enable admins to update passwords of other users? (no problem if not, also not huge priority)

correct not yet done

**Comment by parndt** [comment]
> related to that, please also note [#2 (comment)](https://github.com/dumparkltd/marine-defrag-server/issues/2#issuecomment-946700727)

great, thanks!

**Pull request merged**